src/cf-tables/usage.md

```diff
@@ -1,14 +1,29 @@
 The cf-tables component formats tables, and provides an easy way to make tables sortable.
+The [`cf-core`](../core) component is a dependency of this component."
 
+<<<<<<< b6939a2f8eb00274fc8c71d8c555ae808ec48a9f
```

Comment:
Ah, didn't save my fixes after I when I was rebasing. Fixed in 9e26bfcef6c569b5ba90bc9a0499d722c8fadc68